cmd/nerdctl/images.go

```diff
@@ -132,11 +140,12 @@ func printImages(ctx context.Context, clicontext *cli.Context, client *container
 		}
 	}
 
-	s := client.SnapshotService(clicontext.String("snapshotter"))
+	snapshotter := viper.GetString("snapshotter")
```

Comment:
same comment above is this viper here is able to find `CONTAINERD_SNAPSHOTTER` ?